zerver/tests/test_zephyr.py

```diff
@@ -53,11 +50,11 @@ def logging_mock() -> Any:
         with \
                 ccache_mock(return_value=b'1234'), \
                 ssh_mock(side_effect=subprocess.CalledProcessError(1, [])), \
-                logging_mock() as log:
+                self.assertLogs(level='ERROR') as log:
             result = post("zephyr", cred=cred)
 
         self.assert_json_error(result, 'We were unable to setup mirroring for you')
-        log.assert_called_with("Error updating the user's ccache", stack_info=True)
+        self.assertIn(log.records[0].stack_info, log.output[0])
```

Comment:
This use of `.stack_info` is a bad conversion -- part of the goal of this sort of check is to verify that we get the expected error message, and not some other, and the way you've written it is a tautology that doesn't do that,   I changed this back to use the actual string.